### PR TITLE
TELCODOCS-1675 - Adding updated T-GM config to 4.13+

### PR DESCRIPTION
**Pull request by aireilly**
Adding updated T-GM config to PTP docs

Version(s):
4.13+

Issue:
* https://issues.redhat.com/browse/TELCODOCS-1675

Link to docs previews:
* https://file.emea.redhat.com/aireilly/TELCODOCS-1675/networking/using-ptp.html#configuring-linuxptp-services-as-grandmaster-clock_using-ptp

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by aneeshkp — APPROVED** [review]

File: snippets/ptp_PtpConfigGmWpc.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+apiVersion: ptp.openshift.io/v1
```
Comment (skrthomas) [inline snippets/ptp_PtpConfigGmWpc.yaml:1]: Need to add the snippet metadata:


Every text snippet should be placed in the snippets/ folder and should contain the following metadata at the top:

// Text snippet included in the following assemblies: (1)
//
// * list of assemblies where this text snippet is included
//
// Text snippet included in the following modules:    (2)
//
// * list of modules where this text snippet is included

:_mod-docs-content-type: SNIPPET     

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#text-snippet-file-metadata

File: modules/nw-ptp-configuring-linuxptp-services-as-grandmaster-clock.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [id="configuring-linuxptp-services-as-grandmaster-clock_{context}"]
 = Configuring linuxptp services as a grandmaster clock
 
-You can configure the `linuxptp` services (`ptp4l`, `phc2sys`, `ts2phc`) as grandmaster clock by creating a `PtpConfig` custom resource (CR) that configures the host NIC.
+You can configure the `linuxptp` services (`ptp4l`, `phc2sys`, `ts2phc`) as grandmaster clock (T-GM) by creating a `PtpConfig` custom resource (CR) that configures the host NIC.
```
Comment (skrthomas) [inline modules/nw-ptp-configuring-linuxptp-services-as-grandmaster-clock.adoc:9]:
```suggestion
You can configure the `linuxptp` services (`ptp4l`, `phc2sys`, `ts2phc`) as a grandmaster clock (T-GM) by creating a `PtpConfig` custom resource (CR) that configures the host NIC.
```

Comment (skrthomas) [reply modules/nw-ptp-configuring-linuxptp-services-as-grandmaster-clock.adoc:9]: I see elsewhere "the" precedes grand 
master clock. Maybe an "a" here?

**Review by skrthomas — COMMENTED** [review]
A couple of comments for you :) Nice work!

File: snippets/ptp_PtpConfigGmWpc.yaml (hunk 1 above)
Comment (aireilly) [reply snippets/ptp_PtpConfigGmWpc.yaml:1]: Not for YAML snippets though.

**Pull request merged**